packages/framework/src/Model/Sitemap/SitemapFacade.php

```diff
@@ -80,11 +80,11 @@ public function generateForAllDomains()
      * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domainConfig
      * @return \Shopsys\FrameworkBundle\Model\Sitemap\SitemapItem[]
      */
-    public function getSitemapItemsForVisibleProducts(DomainConfig $domainConfig)
+    public function getSitemapItemsForListableProducts(DomainConfig $domainConfig)
```

Comment:
Renaming of public method in frameworks means you are introducing BC break (if anyone extended this method in theirs project, this will cause their application to fail on non existing method). This change should be mentioned in upgrade notes so developers are informed about this and can change theirs application accordingly. Same for SitemapRepository:: getSitemapItemsForListableProducts

Reply:
ok, i add upgrade note